test/support/api/dashboards.js

```diff
@@ -1,10 +1,20 @@
 import _ from 'underscore';
-import { getResource } from 'helpers/json-api';
+import { v4 as uuid } from 'uuid';
+
+import { getResource, mergeJsonApi } from 'helpers/json-api';
 
 import fxTestDashboards from 'fixtures/test/dashboards';
 
 const TYPE = 'dashboards';
 
+export function getDashboard(data) {
+  const resource = getResource(_.sample(fxTestDashboards), TYPE);
+
+  data = _.extend({ id: uuid() }, data);
+
+  return mergeJsonApi(resource, data);
+}
+
 Cypress.Commands.add('routeDashboards', (mutator = _.identity) => {
```

Comment:
We could use `getDashboard()` in the `routeDashboards` command as well, I think? With a sample of `3`?

Reply:
I think that's probably correct

Reply:
Made this update ☝️

Reply:
It didn't work, reverted it back for now.

Reply:
Okay, I got some working for this. In latest commit.